### PR TITLE
nathelper: add optional set_contact_alias([trim]) parameter

### PR DESCRIPTION
**Pull request by eschmidbauer**
#### Pre-Submission Checklist
- [x] Commit message has the format required by CONTRIBUTING guide
- [x] Commits are split per component (core, individual modules, libs, utils, ...)
- [x] Each component has a single commit (if not, squash them into one commit)
- [x] No commits to README files for modules (changes must be done to docbook files
in `doc/` subfolder, the README file is autogenerated)

#### Type Of Change
- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)

#### Checklist:

- [ ] PR should be backported to stable branches
- [x] Tested changes locally
- [x] Related to issue #2308 

#### Description
add support for trimming an already existing alias parameter when calling `set_contact_alias()`

### CODE REVIEW

File: src/modules/nathelper/nathelper.c [hunk 1]
```diff
@@ -704,10 +707,12 @@ static int fix_nated_contact_f(struct sip_msg *msg, char *str1, char *str2)
  * Replaces ip:port pair in the Contact: field with the source address
  * of the packet.
  */
-static int set_contact_alias(struct sip_msg *msg)
+static int set_contact_alias(struct sip_msg *msg, int trim)
```
Comment (miconda) [inline src/modules/nathelper/nathelper.c:710]: By changing the signature of `set_contact_alias()` you broke the kemi export of it.

Comment (eschmidbauer) [reply src/modules/nathelper/nathelper.c:710]: > By changing the signature of `set_contact_alias()` you broke the kemi export of it.

sorry i haven't worked with the kemi export code yet. i pushed another commit which i _think_ fixes it

**Comment by miconda** [comment]
Not really fixing it, because now set_contact_alias() from kemi require a parameter always, so existing kemi deployement will be broken on upgrade. A new function can be exported to kemi (e.g., set_contact_alias_trim()) and the old one should stay the same. So practically a new wrapper function has to be created to cope with the two cases.

**Comment by eschmidbauer** [comment]
> Not really fixing it, because now set_contact_alias() from kemi require a parameter always, so existing kemi deployement will be broken on upgrade. A new function can be exported to kemi (e.g., set_contact_alias_trim()) and the old one should stay the same. So practically a new wrapper function has to be created to cope with the two cases.

thanks for the guidance on that.
 i think latest commit takes care of it correctly.

**Comment by miconda** [comment]
Pretty much, I would only suggest to use `ki_` prefix for the kemi functions, like the other wraperss which have a specific prefix for exports to kemi (e.g., the last three in the kemi exports structure).

**Comment by miconda** [comment]
If nobody else will have opinions against, then I will merge this PR before branching 5.4

**Pull request merged**